packages/mux-video/src/index.ts

```diff
@@ -359,7 +359,8 @@ class MuxVideoElement extends CustomVideoElement<HTMLVideoElement> implements Pa
         if (newValue === oldValue) {
           break;
         }
-        this.__updateAutoplay?.(newValue);
+        /** In case newValue is an empty string or null, use this.autoplay which translates to booleans (WL) */
+        this.__updateAutoplay?.(this.autoplay);
```

Comment:
FYI `__updateAutoplay()` didn't handle the case when newValue was an empty string